src/app/pages/account/users/user-form/user-form.component.ts

```diff
@@ -435,4 +443,15 @@ export class UserFormComponent {
 
     return username.toLocaleLowerCase();
   }
+
+  private updateShellOptions(group: number, groups: number[]): void {
+    let ids: number[] = [];
```

Comment:
Alternatively, `new Set()` can fit better here to avoid duplicate values when adding the group id below.